FI.API.SignTool/FI.API.SignTool.csproj

```
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp2.2</TargetFrameworks>
```

[2.1 is LTS and 3.0 is latest]()https://dotnet.microsoft.com/platform/support/policy/dotnet-core
```suggestion
    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
```

---

Removed multi targeting